src/controllers/userProfileController.js

```diff
@@ -51,6 +51,7 @@ async function ValidatePassword(req, res) {
 }
 
 const userProfileController = function (UserProfile) {
+  console.log('Hello')
```

Comment:
Should remove console.log statement.

Reply:
@ptrpengdev I went through and removed the remaining console logs that you mentioned :) Please let me know if i missed anything else

Reply:
lgtm. Please do remember to pull the latest code from the development branch and resolve any conflicts. Thanks!